usagov/vote-gov-drupal
# VOTE-2597: Upgrade to PHP 8.3

## Jira ticket
[VOTE-2597](https://cm-jira.usa.gov/browse/VOTE-2597)

## Description

Upgrade environments to php 8.3.

## Deployment and testing

### Post-deploy steps

1. Locally run `lando rebuild` (if needed run `lando restart` too)

### QA/Testing instructions

1. Login to the site and go to Status Report page and confirm that the PHP version is 8.3.

## Checklist for the Developer




## Checklist for the Peer Reviewers
